.yamato/config/__shared.metafile

```diff
@@ -143,9 +153,8 @@ platforms:
     apis:
       - name: Metal
         exclude_test_platforms:
-        - editmode
```

Comment:
wondering the same thing about editmode

Reply:
because the performance build tests are editmode tests